stopwatch.py

```diff
@@ -346,6 +351,24 @@ def format_last_report(self):
     # Private methods
     #################
 
+    def _pop_stack(self, name, end_type='end'):
```

Comment:
Can you update this docstring to include the arguments. It looks like the end_type argument is only used in the assertion message? Can we just re-check the cancelled_spans to figure out the assertion message and avoid the extra arg?